core/iwasm/aot/aot_loader.c

```diff
@@ -1743,7 +1743,7 @@ load_from_sections(AOTModule *module, AOTSection *sections,
                     export_tmp = module->exports;
                     for (j = 0; j < module->export_count; j++, export_tmp++) {
                         if ((export_tmp->kind == EXPORT_KIND_FUNC)
-                            && (!strcmp(export_tmp->name, "__retain"))) {
+                            && (!strcmp(export_tmp->name, "__pin"))) {
```

Comment:
There might be someone who is using the older asc compiler, we had better be compatible with the older version, could you please changing it to:
```C
&& (!strcmp(export_tmp->name, "__retain")
    || !strcmp(export_tmp->name, "__pin"))
```

Reply:
Thank you for your thoughtful consideration, I'll change it and test it again.